lib/index.js

```diff
@@ -71,6 +71,9 @@ export default function runContentfulExport (usageParams) {
           }
         )
       }
+      return response
```

Comment:
Another item for the future could be to extract logic to separate functions so its easier to digest